analyzer/src/main/kotlin/curation/ClearlyDefinedPackageCurationProvider.kt

```diff
@@ -110,6 +118,14 @@ class ClearlyDefinedPackageCurationProvider(server: Server = Server.PRODUCTION)
 
             log.warn { "Deserializing the ClearlyDefined curation for '${pkgId.toCoordinates()}' failed." }
 
+            return emptyList()
+        } catch (e: Exception) {
```

Comment:
Currently we only experience SocketTimeoutExceptions. 

Might it not be better to start with a specific exception that actually occurs and catch it, instead of all?
It might happen that other, important exceptions are overlooked, when a user becomes accustomed to the warning.

Reply:
In this special case, I see the problem that any unexpected exception will cause the analyzer to crash. For a relatively unimportant task like loading optional curations from ClearlyDefined, this should not be the case IMHO.